source/dhtslib/tabix.d

```diff
@@ -108,12 +116,12 @@ struct TabixIndexedFile {
             // but first row was preloaded in this.next)
             private bool active;
 
-            this(htsFile *fp, tbx_t *tbx, const(char)[] r)
+            this(htsFile *fp, tbx_t *tbx, string chrom, Zbho coords)
             {
                 this.fp = fp;
                 this.tbx= tbx;
 
-                this.itr = tbx_itr_querys(tbx, toStringz(r) );
+                this.itr = tbx_itr_queryi(tbx, tbx_name2id(tbx, toStringz(chrom)), cast(int) coords.start, cast(int) coords.end);
```

Comment:
cast(int) ? I thought htslib was now on longs -- is this an oversight on my part (bindings) ?

Reply:
Actually we are now up to htslib version 1.12

Reply:
Note to self see #35

Reply:
I pushed a branch tonight after running dstep from hstlib 1.12. I cleaned up some of it but there are a good amount of changes that are just formatting.

Reply:
So will this PR build on that branch? i.e. do we need to update this line with a new commit before I resolve adn accept?

Reply:
I am not sure. I plan to rework branch htslib112 to add in the changes we discussed today and address htslib_endian. We could merge htslib112 into this branch before accepting.

Reply:
I have reworked the branch so if you agree I can merge it into this branch.

Reply:
Additionally I have another branch with other filetype readers that is related. Though we can make that a separate pull request if you would like.